express/index.cjs

```diff
@@ -6,8 +6,7 @@ const bcrypt = require('bcrypt');
 const bodyParser = require('body-parser');
 const fs = require('fs');
 const { rateLimit } = require('express-rate-limit')
-const {checkUser, checkAuth, login, setTimestamp, validSAN, validSA, validA, validN, validState, validJSON, validExpDate, validDate, validDates} = require('./helper.js');const { title } = require('process');
-const { sendEmail } = require('./Email.js');
+const {checkUser, checkAuth, login, setTimestamp, validSAN, validSA, validA, validN, validState, validJSON, validExpDate, validDate, validDates} = require('./helper.js');const { title } = require('process');const { sendEmail } = require('./Email.js');
```

Comment:
I'm just wondering if the title import is still needed since it's not read anywhere besides it being used in the queries